main.py

```diff
@@ -178,19 +178,11 @@ async def compile_project(project_root: Path, manifest: PackageManifest):
     # Create a contracts directory
     contracts_dir = project_root / "contracts"
     contracts_dir.mkdir()
+    project = Project.from_manifest(manifest)
```

Comment:
btw when it come time to add the dependencies, you can do it like this:

```python
project = Project.from_manifest(manifest, config_override={"dependencies": [{"pypi": "snekmate"}]})
```

Reply:
can `config_override` accept a list of `DependencyAPI` objects? that's how I was thinking to parse the incoming dependency specifications

Reply:
i think you can pass in objects as well but you dont have to: itll validate them if you use raw dicts